build.gradle

```diff
@@ -1,9 +1,18 @@
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
Supposedly `jcenter` is a superset of `mavenCentral` and has been the **default** for new Android projects for at least a year now. Some people claim it's faster in some regions because of a better CDN. I've personally not had any issue with either. Do you want me to change it back to `mavenCentral`?